fl_training.py

```diff
@@ -138,8 +138,8 @@ def compute_test_score(node_list):
                           verbose=0,
                           validation_data=(node.x_val, node.y_val))
                 
-                val_acc_list.append(history.history['val_acc'])
-                acc_list.append(history.history['acc'])
+                val_acc_list.append(history.history['val_accuracy'])
+                acc_list.append(history.history['accuracy'])
```

Comment:
For my culture, is this a Keras usage best practice?

Reply:
Sorry in the first version that I used to test 'acc' did not work. I needed to write 'accuracy' and 'val_accuracy' (same for Samya). But I had to install again everything and now it works with 'acc'.